### ISSUE TITLE
[Verification] Unsupported/missing verification

### ISSUE THREAD

**Issue by squakez** [syndesisio/syndesis#8608]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
While working on #7466 I noticed that the `UI` is not treating a verification `unsupported` status properly, showing any unsupported verification as `validated` instead:

![unsupported_verification](https://user-images.githubusercontent.com/23169723/83391532-e5712780-a3f3-11ea-8d30-fbd302c37ef0.png)

The verification endpoint returns:
```
[{"status":"UNSUPPORTED","scope":"PARAMETERS","errors":[{"code":"unknown-connector","description":"No connector for ID i-M8PdK9xYED-7ijVaUwYz registered"}]}]
```
But the `UI` is showing as verified correctly. I suggest to either show an error or a warning mentioning the verification is not supported for that connector.

**Comment by gashcrumb** [comment]
Yeah, looks like currently the UI is only [concerned with an ERROR response](https://github.com/syndesisio/syndesis/blob/master/app/ui-react/syndesis/src/modules/connections/utils.ts#L9).  We could probably add a case for UNSUPPORTED but I don't know if I'd expose that error message as it is to the user.

**Comment by gashcrumb** [comment]
I guess if a connector doesn't support validation we should hide/disable the "Validate" button instead.

**Comment by squakez** [comment]
> I guess if a connector doesn't support validation we should hide/disable the "Validate" button instead.

I agree @gashcrumb. The problem I can see right now is how we can drive `UI` on that direction as the configuration to enable a validation process is not exposed through the API. I have opened another issue #8609 to take care of this particular situation, we may evaluate how to provide such configuration while working on it.

However I think we still have to provide a failsafe mechanism to cover any other response code that is not a real success validation. We probably won't have to give details to the user, just reporting that the validation process is not enabled for that connector.
